### DISCUSSION TITLE
FEATURE REQUEST - QR Code Generator Logic For Impossible Options

### DISCUSSION THREAD

**Discussion by michaelmcdonald** [gopro/labs#438] [Feature Requests]
Noticed that I could create a QR code that setups up 5k 8:7 video resolution and 240 FPS Video. This is obviously a combination that cannot occur. My recommendation is a set of logical "gates" that allow / disallow options based on options chosen.

**Comment by dnewman-gpsw** [comment]
This was by design, when the QR code language was initially developed, it was for HERO5, it only had 4K30, yet the commands for 4K60 and 4K120 existed, now they work on today's cameras.  Adding gates is just something that future cameras will break, and I don't like making future busy work.  As these webpages are all open source, any user can create variations that include gates if they wish.
